architecture/pricing-sources-matrix.md

```diff
@@ -0,0 +1,75 @@
+| [Cloud Provider-Billing Integrated](/install-and-configure/install/cloud-integration/README.md) | Kubecost pulls cloud provider billing every 6 hours<br>When billing becomes available Kubecost reconciles the previously-estimated costs | Extremely accurate<br>Little maintenance<br>All cloud billing can be imported<br>Out-of-cluster costs can be [combined with Kubernetes resources](/using-kubecost/navigating-the-kubecost-ui/collections.md) | Kubecost requires access to the billing account<br>This setup can take time, especially if the team deploying Kubecost does not have access to the billing account |
+| [CSV Pricing](/install-and-configure/advanced-configuration/csv-pricing.md) | Kubecost Enterprise can use a user-provided CSV with granular resource prices. | Does not require cloud provider access<br>Works for on-prem clusters<br>Completely air-gapped for highly-secure environments | Mapping labels in the CSV can be tedious |
+| [AWS Spot Data Feed](/install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-spot-instances.md) | Kubecost pulls spot prices from a custom feed | More accurate costs in the short-term (48 hours) for Spot nodes | Limited value if billing-integration is configured<br>Requires additional setup with AWS |
+| [Azure Rate Card](../install-and-configure/install/cloud-integration/azure-out-of-cluster/azure-config.md) | Kubecost pulls Azure Rate Card API | More accurate Azure costs in the short-term (48 hours) | Limited value if billing-integration is configured<br>Requires additional setup with Azure |
+| [Alibaba](../install-and-configure/install/provider-installations/alibaba-install.md) | Kubecost pulls Alibaba Rate Card API | Currently the only method to retrieve Alibaba resource pricing |More accurate costs<br>Requires additional setup with Alibaba |
+
+## Diagram
+
+![Cloud Provider Billing Integrated](../images/cloud-bill-diagram.png)
+
+## Details
+
+### Cloud Provider OnDemand API
+
+- [Alibaba](../install-and-configure/install/provider-installations/alibaba-install.md)
+- [AWS EC2](https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/us-east-2/index.json)
+- [Azure Rate Card](../install-and-configure/install/cloud-integration/azure-out-of-cluster/azure-config.md)
+
+### Custom Pricing
+
+Set via your *values-kubecost.yaml*, or in the Kubecost UI by going to *Settings*, then under 'Pricing', toggling on 'Enable Custom Pricing', then making adjustments per group:
+
+![UI Custom Pricing Screenshot](/images/custompricing.png)
+
+Prices are monthly. `storage` and miscellaneous network metrics are all per GB.
+
+```yaml
```

Comment:
Should we call out that these are the helm values?
@bstuder99 Can we just name the code block as the example `values-kubecost.yaml` mentioned above?